Musicplayer.css

```diff
@@ -134,29 +165,28 @@ ul li ul li:hover{background: black}
 	width: 100%;
 	display: grid;
     grid-gap: 2%;
-	grid-template-columns: 2% 14% 14% 14% 14% 14% 14%;
+	grid-template-columns: 6% 14% 14% 14% 14% 12% 12%;
     grid-template-rows: 70px;
-	padding-top: 4px;
-	padding-left:5%;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
 }
 
 .content_4{
 	grid-column-start: 1;
 	grid-column-end: span 4;
 	background-color: white;
 	z-index: 2;
-	height:600px;		
+	height:600px;
```

Comment:
I also set the font for it and color (both background and text color) and it will be back and white which display a great contrast for user so they can easy read the description.(apply element in the music database to make it consistence)